### DISCUSSION TITLE
Section 3: Lesson 3: Decoding Ethereum transactions-->constants.js: contractAddress

### DISCUSSION THREAD

**Discussion by Faridiberraken** [Cyfrin/foundry-full-course-cu#1761] [Q&A]
In the HTML-FUND-ME-F23 project, the constants.js file contains a hardcoded value for contractAddress.
How is it possible that every time we deploy the FundMe contract on Anvil, we get the same contract address?
Tks!

**Comment by Faridiberraken** [comment]
I close the issue, it is already answered in: [https://github.com/Cyfrin/foundry-full-course-f23/discussions/1313#discussion-6058673](url)

**Reply by EvolOfThings** [reply]
@Faridiberraken the link is dead.

**Reply by EngrPips** [reply]
Hello @EvolOfThings, Don't click on the link. Instead, copy and paste It into your browser. It works for me that way

<img width="1440" alt="Screenshot 2024-07-09 at 05 22 33" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/95760918/82bec3c0-3296-4471-9854-a961ddebd3db">

**Reply by EvolOfThings** [reply]
Oh yes! it works, weird! Thanks ser! 🫡

**Reply by EngrPips** [reply]
Awesome, You are welcome.
